src/pyroe/make_txome.py

```diff
@@ -633,8 +634,10 @@ def make_splici_txome(
     # get introns
     # the introns() function uses inplace=True argument from pandas,
     # which will trigger an FutureWarning.
-    # warnings.simplefilter(action="ignore", category=FutureWarning)
+    warnings.simplefilter(action="ignore", category=FutureWarning)
```

Comment:
This is good for now, but do we want to post an issue in the pyranges GitHub about this so they can fix the issue upstream?  I don't want to end up in a situation where it *breaks* and then we aren't aware.